Analysis/src/AstJsonEncoder.cpp

```diff
@@ -1136,6 +1138,31 @@ struct AstJsonEncoder : public AstVisitor
         );
     }
 
+    void write(AstAttr::Type type)
+    {
+        switch (type)
+        {
+        case AstAttr::Type::Checked:
+            return writeString("checked");
+        case AstAttr::Type::Native:
+            return writeString("native");
+        default:
+            return writeRaw("null");
```

Comment:
```suggestion
```
Best not to include defaults for enum switches. If the compiler cannot verify exhaustiveness, assert can be used.